File: Android/SahilBhragudev_SahilBhragudev_2024cse1116_2/GettingStarted/prime_till_n.java
import java.util.*;
public class prime_till_n {
    public static void main(String[] args) {
        // write your code here
        Scanner sc = new Scanner(System.in);
        
        int low , high;
        System.out.println("Enter the lower range");
        low = sc.nextInt();
        System.out.println("Enter the higher range");
        high = sc.nextInt();

        
        for(int num=low;num<=high;num++)
        {  int count=0;
            for(int i=1;i<=num;i++)
            {    
               if(num%i==0)
                 {
                   count++;
                 }
            }
            
              if(count==2)
                   {
                      System.out.println(num);
                   }
                  
        }
      
    }
}

File: ML/palprashant2002_PrashantKumarPal_2024CSE1146_2/GettingStarted/Prime_till_N.cpp
#include<iostream>
using namespace std;
int main()
{
    int n,i,num,c;
    cout<<"Enter N :";
    cin>>n;
    for(num=2;num<=n;num++)
    {
        c=0;
        for(i=2;i<(num/2)+1;i++)
        {
            if(num%i==0)
            {
                c=1;
                break;
            }
        }
        if(c==0)
        {
            cout<<num<<" Prime\n";
        }
    }
    return 0;
}